### ISSUE TITLE
move bugs

### ISSUE THREAD

**Issue by zoli** [limetext/commands#4]
Some issues are fixed after #2 but there are still other bugs here that we can see on qml frontend.

**Comment by ricochet1k** [comment]
More detail: 
- Moving up into a line with less characters than the current cursor position actually moves left some number instead of moving to the end of the line
- Moving down into a line with less characters than the current cursor position seems to move x characters after the start of the next line, where x is the current cursor column

I don't know where this needs to go, but another thing to remember is that _in most editors_ if you are moving up and down between lines with different lengths, the cursor column is remembered and the cursor is placed as close to it as possible, as long as only up and down arrows are pressed.

**Comment by ricochet1k** [comment]
#6 fixes at least part of this bug. I don't know if there are more movement bugs.

**Comment by zoli** [comment]
I don't see any issues with the move command currently. 
I'll close this issue, We should open another issue for each other bug with explicit detail.
